build-tools-internal/src/main/groovy/elasticsearch.build-complete.gradle

```diff
@@ -12,7 +12,9 @@ import java.nio.file.Files
 
 String buildNumber = System.getenv('BUILD_NUMBER') ?: System.getenv('BUILDKITE_BUILD_NUMBER')
 String performanceTest = System.getenv('BUILD_PERFORMANCE_TEST')
-if (buildNumber && performanceTest == null && GradleUtils.isIncludedBuild(project) == false) {
+Boolean isNested = System.getProperty("scan.tag.NESTED") != null
+
+if (buildNumber && performanceTest == null && GradleUtils.isIncludedBuild(project) == false && !isNested) {
```

Comment:
nit: let's at least consistently use `!` vs `== false` in the same conditional. I don't have strong opinions on which although we use `== false` conventionally in the ES codebase.